client/js/screens/Profile/Profile.js

```diff
@@ -1,29 +1,84 @@
 import React from "react";
-import { View, Text, TouchableOpacity } from "react-native";
+import { View, Text, TouchableOpacity, Image } from "react-native";
 import { logOut } from "../../config/models";
 import PropTypes from "prop-types";
+import { styles } from "./styles";
+import theme from "../../config/styles";
+import * as Progress from "react-native-progress";
 
-const Profile = ({ navigation }) => {
+const Profile = ({ navigation, user }) => {
+  const maxXp = 4000;
+  const currentXp = 600;
   return (
-    <View>
-      <TouchableOpacity
-        onPress={async () => {
-          try {
-            logOut();
-            navigation.navigate("Auth");
-          } catch (e) {
-            return false;
-          }
-        }}
-      >
-        <Text>Log Out</Text>
-      </TouchableOpacity>
+    <View style={styles.root}>
+      <View style={styles.userInfoContainer}>
+        <TouchableOpacity style={styles.editButton}>
+          <Text style={styles.editButtonText}>Edit</Text>
+        </TouchableOpacity>
+        <Image
+          style={styles.backgroundImage}
+          source={require("../../assets/images/profile-background.png")}
+        />
+        <Image style={styles.userAvatar} source={{ uri: user.image }} />
+        <View style={styles.userInfo}>
+          <Text style={styles.userName}>{user.name}</Text>
+          <Text style={styles.userEmail}>{user.email}</Text>
+        </View>
+      </View>
+      <View style={styles.userStatsContainer}>
+        <View style={styles.userStatsHeaderContainer}>
+          <Text style={styles.userStatsHeader}>Level 1: NoviceLearner</Text>
+          <Image
+            style={styles.headerImage}
+            source={require("../../assets/images/trophies-banner.png")}
+          />
+        </View>
+
+        <View style={styles.userProgressContainer}>
+          <Text style={styles.userProgressText}>
+            {maxXp - currentXp} xp until you react the next level!
+          </Text>
+          <Progress.Bar
+            progress={currentXp / maxXp}
+            style={styles.progressBar}
+            width={200}
+            height={16}
+            color={theme.colors.seaBlue}
+            borderRadius={10}
+          />
+          <Text style={styles.userProgressText2}>
+            {currentXp}/{maxXp}xp
+          </Text>
+        </View>
+        <View style={styles.userStatsLessonsContainer}>
+          <Text style={styles.userStatsLessonsText}>Lessons Completed:</Text>
+          <Text style={styles.userStatsLessons}>0</Text>
```

Comment:
Will this value be pulled from your database to check how many lessons a certain logged in user has completed?